sound/hybrid_font.h

```diff
@@ -120,7 +128,37 @@ class HybridFont : public SaberBase {
       PlayPolyphonic(effect);
     }
   }
+  
+  void StartSwing() override {
+    if (!guess_monophonic_) {
+      if (!swing_player_) {
+        swing_player_ = PlayPolyphonic(&swng);
+      }
+      // avoid overlapping swings unless at least a quarter of the swing has played.  Stops double swing events.
+      if ((swing_player_->length() - swing_player_->pos()) / swing_player_->length() <= 0.75) {
+        RefPtr<BufferedWavPlayer> overlap_swing = swing_player_;
+        swing_player_ = PlayPolyphonic(&swng);
+        overlap_swing->set_fade_time(0.1);
```

Comment:
I'm not sure this is what we want to do.
Previously, swing sounds were just left to play out by themselves.

Reply:
This only applies for rapid swings playing over each other, like spins. The idea is to allow rapid accent swing sounds to play over each other without having to worry about each swing sound and setting volume for each. For normal saber swinging, this doesn’t really factor in.

Reply:
we could dynamically set the fade time to the remaining duration of the swing.  That way all we're really doing is a cross fade.  I'll give that a try.

Reply:
does overlap_swing->set_fade_time(overlap_swing->length() - overlap_swing->pos()); look good for a dynamic fade time, based on the remaining play time of an effect?

Reply:
removed fade time

Reply:
Put fade time back in.  During testing, odd cutoffs occurred without it.